Objects/classobject.c

```diff
@@ -193,34 +203,32 @@ method_getattro(PyObject *obj, PyObject *name)
     return PyObject_GetAttr(im->im_func, name);
 }
 
-PyDoc_STRVAR(method_doc,
-"method(function, instance)\n\
-\n\
-Create a bound instance method object.");
+/*[clinic input]
+@classmethod
+method.__new__
```

Comment:
```suggestion
method.__new__ as method_new
```
A bit weird to have "method" as a function name in C. Same for instancemethod below.